tools/flang2/flang2exe/ll_structure.c

```diff
@@ -591,11 +591,14 @@ ll_create_module(const char *module_name, const char *target_triple,
 
 void
 add_linker_directives(LLVMModuleRef module) {
+#if LLVM_VERSION_MAJOR < 5
    LLMD_Builder mdb = llmd_init(module);
    char* linker_directive;
    for (int i = 0; (linker_directive = flg.linker_directives[i]); ++i) {
        LLMD_Builder submdb = llmd_init(module);
-
+       if (strlen(linker_directive) > 2 && linker_directive[0] == '\\' &&
+           linker_directive[1] == '-')
```

Comment:
Major hack to support, `-linker -defaultlib:flangrti`. Basically `flang.exe` sends `-linker \-defaultlib:flangrti`

Reply:
The way `cl.exe` does this is that all arguments after `-link` are simply passed down to the linker. So in the same way, we could just have a `-link` and then everything after it is just passed directly to the linker.

Reply:
What do you think?

Reply:
I don't want to make a big change like that without talking with upstream.

Reply:
Let's talk now. Over to slack.

Reply:
I reverted this change for now